pkg/controllers/validationController_test.go

```diff
@@ -106,7 +106,7 @@ func TestValidateRequestBodyWithNotAllowedK8sResource(t *testing.T) {
 	validationController := NewValidationController()
 	validationController.Validate(responseRecorder, request)
 
-	assert.Contains(t, strings.TrimSpace(responseRecorder.Body.String()), "\"allowed\":false")
+	assert.Contains(t, strings.TrimSpace(responseRecorder.Body.String()), "\"allowed\":true")
```

Comment:
The title of the unit test says "not allowed resource", why should it be allowed?